roles/edpm_accel_drivers/tasks/boot_config.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Check if grub2-mkconfig has --update-bls-cmdline option
+  ansible.builtin.shell:
+    cmd: grub2-mkconfig --help | grep '\-\-update-bls-cmdline'
+  ignore_errors: true
+  register: _edpm_accel_drivers_check_update_bls_cmdline
+  changed_when: false
+
+- name: Regenerate initramfs
+  become: true
+  ansible.builtin.command: "{{ item }}"
+  loop:
+    - 'dracut --force'
+    - >-
+      grub2-mkconfig -o /boot/grub2/grub.cfg
+      {{ '--update-bls-cmdline'
+      if _edpm_accel_drivers_check_update_bls_cmdline.rc == 0
+      else '' }}
+  changed_when: true
```

Comment:
Why do you need changed_when: true here?

Reply:
good point. the original code from the arch DT was:
```
        - name: Check if grub2-mkconfig has --update-bls-cmdline option
          ansible.builtin.shell:
            cmd: grub2-mkconfig --help | grep '\-\-update-bls-cmdline'
          ignore_errors: true
          register: check_update_bls_cmdline
          changed_when: false

        - name: Regenerate initramfs
          become: true
          ansible.builtin.command: "{{ item }}"
          loop:
            - 'dracut --force'
            - >-
              grub2-mkconfig -o /boot/grub2/grub.cfg
              {{ '--update-bls-cmdline'
              if check_update_bls_cmdline.rc == 0
              else '' }}
          when: (_blacklist_nvidia.changed or _load_vfio.changed)
```